src/ABBAurora.cpp

```diff
@@ -19,7 +19,11 @@ void ABBAurora::setup(HardwareSerial &hardwareSerial, byte RXGpioPin, byte TXGpi
     digitalWrite(TXPinControl, LOW);
 
     serial = &hardwareSerial;
-    serial->begin(19200, SERIAL_8N1, RXGpioPin, TXGpioPin, false, 500);
+    #if defined(ESP32)
```

Comment:
1. Removing support for Atmel arduinos, not good idea
2. if we define PINs based on  platform, perhaps we don't need to make different calls to setup. Clean code, better readability, improve maintainability